.github/workflows/js_tests.yml

```diff
@@ -0,0 +1,23 @@
+name: JS Tests
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
```

Comment:
This should maybe also include `master`, for the releases?
I understand that `master` and `develop` should contain similar stuff, but in the case of hotfixes they wouldn't

Reply:
Should i do all pull and pushes actually? `on: [push, pull_request]`

Reply:
Sure - that would do it on all the branches, correct?
I think that makes sense 👍

Reply:
Fixed.